cmake/scipp-install.cmake

```diff
@@ -10,7 +10,11 @@ function(scipp_install_component)
   )
   add_sanitizers(${SCIPP_INSTALL_COMPONENT_TARGET})
   if(DYNAMIC_LIB)
-    install(TARGETS ${SCIPP_INSTALL_COMPONENT_TARGET} EXPORT ${EXPORT_NAME})
+    install(
+      TARGETS ${SCIPP_INSTALL_COMPONENT_TARGET}
+      EXPORT ${EXPORT_NAME}
+      RUNTIME DESTINATION Lib/scipp
```

Comment:
Is this flag used only by Windows? Otherwise it would break OSX and Linux?

Reply:
According to https://cmake.org/cmake/help/latest/command/install.html it affects executables (we don't install any as far as I can see) and DLLs (only on windows systems). On linux and osx I believe cmake uses `LIBRARY DESTINATION ...`